en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/S/Sunburst.md

```diff
@@ -9,7 +9,8 @@ _8th-level evocation_
 **Duration:** Instantaneous
 
 Oozes and undead have disadvantage on saving throws made to resist this spell.
-You create a burst of radiant sunlight that fills the area. Each creature in the area takes 12d6 radiant damage and is blinded for 1 minute.
+You create a burst of radiant sunlight that fills the area. Each creature in the area takes 12d6 radiant damage and is blinded for 1 minute if it fails a Constitution saving throw.
```

Comment:
Huh, I missed both the line break and the damage highlighting here... It should be
```markdown
You create a burst of radiant sunlight that fills the area.
Each creature in the area takes `12d6` radiant damage and is blinded for 1 minute if it fails a Constitution saving throw.
```
with your changes. (Followed by the line about saving throw success, which is great.)

Reply:
Still missing a line break in line 13. 😛 (Before _"On a successful save"_.)